examples/react/src/app/services/counter.ts

```diff
@@ -35,3 +35,5 @@ export const counterApi = createApi({
     }),
   }),
 });
+
+export const { useDecrementCountMutation, useGetCountQuery, useIncrementCountMutation } = counterApi.hooks;

```

Comment:
@markerikson this is what you have been waiting for, haven't you? ;)

Reply:
niiiiice :)